src/templates/en/2019/contributors.html

```diff
@@ -26,31 +26,63 @@
   border-radius: 10px;
   cursor: pointer;
   margin: 8px;
-  padding: 8px;
+  padding: 8px 16px;
+  border: none;
+  -webkit-font-smoothing: antialiased;
 }
 
 .contributor-grid {
   display: flex;
   align-items: stretch;
+  flex-direction: row;
```

Comment:
On mobile, contributors should appear one per row.

![image](https://user-images.githubusercontent.com/1120896/67813970-3fce9200-fa79-11e9-9a53-ea10fa655814.png)